src/core/main.c

```diff
@@ -3130,11 +3130,6 @@ int main(int argc, char *argv[]) {
                 /* clear the kernel timestamp, because we are not PID 1 */
                 kernel_timestamp = DUAL_TIMESTAMP_NULL;
 
-                /* Clear ambient capabilities, so services do not inherit them implicitly. Dropping them does
-                 * not affect the permitted and effective sets which are important for the manager itself to
-                 * operate. */
-                capability_ambient_set_apply(0, /* also_inherit= */ false);
```

Comment:
This affects other processes forked from the manager that are not sd-executor - generators for example - so I'd keep this here too

Reply:
Hmm, that would defeat the purpose of this PR? We need to retain the capability in manager so that it's inherited by sd-executor?

Reply:
Right, then the safe_fork stuff should start dropping them instead I guess?

Reply:
I won't be too concerned about generators. If they're compromised, malicious unit files can be installed anyway. And we don't fork off any other helper processes that exec()s other executables in manager.

Reply:
Yeah, but as a measure to prevent errors, this was useful. I would prefer to retain the drop for generators too, to avoid any potential for regression.

Reply:
I kind of felt this won't be _that_ easy (-; Improving.

Reply:
I'd rather avoid potential behaviour changes this late. It should be quite straightforward to do this in `execute_strv()`?